code/__HELPERS/unsorted.dm

```diff
@@ -1496,3 +1496,11 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			else
 				return "\[[url_encode(thing.tag)]\]"
 	return "\ref[input]"
+
+// Makes a call in the context of a different usr
+// Use sparingly
+/proc/PushUsr(mob/M, datum/callback/CB)
```

Comment:
World proc, an admin could use this to spoof another admin if I am getting my cards right.